conformance/03_blob_upload_monolithic_test.go

```diff
@@ -18,6 +18,25 @@ var test03BlobUploadMonolithic = func() {
 			Expect(resp.StatusCode()).To(Equal(http.StatusNotFound))
 		})
 
+		g.Specify("POST request with digest and blob should yield a 201", func() {
+			req := client.NewRequest(reggie.POST, "/v2/<name>/blobs/uploads/").
+				SetHeader("Content-Length", configContentLength).
+				SetHeader("Content-Type", "application/octet-stream").
+				SetQueryParam("digest", configDigest).
+				SetBody(configContent)
+			resp, err := client.Do(req)
+			Expect(err).To(BeNil())
+			Expect(resp.StatusCode()).To(Equal(http.StatusCreated))
+			lastResponse = resp
+		})
+
+		g.Specify("GET request to blob URL should yield 200", func() {
+			req := client.NewRequest(reggie.GET, lastResponse.GetRelativeLocation())
```

Comment:
Do we need to use `lastResponse.GetRelativeLocation()` here? This seems to be the session URL. I think this should just be a normal blob GET no?

Reply:
It's a convenience.  The Location header after an upload contains the location of the blob, though it can be accessed by its digest as well.  I changed it to the latter to make it look more like the spec.